packages/connection/src/Manager.php

```diff
@@ -1735,4 +1735,23 @@ public function sign_role( $role, $user_id = null ) {
 
 		return $role . ':' . hash_hmac( 'md5', "{$role}|{$user_id}", $token->secret );
 	}
+
+	/**
+	 * Some hosts disable the OpenSSL extension and so cannot make outgoing HTTPS requsets
+	 */
+	public static function fix_url_for_bad_hosts( $url ) {
+		if ( 0 !== strpos( $url, 'https://' ) ) {
+			return $url;
+		}
+
+		switch ( JETPACK_CLIENT__HTTPS ) {
```

Comment:
When working with constants from a package, it's recommended that we use the `Constants` package for that. Not only this makes our code consistent and independent, but it also protects against fatals.

Reply:
See a142b9c. The Constants package was used for the `JETPACK_CLIENT__HTTPS` constant.